### PR TITLE
Feature/schema changes

### PR DESCRIPTION
**Pull request by HarisShahidHameed**
Trello links : https://trello.com/c/nOi31Cvt/33-changes-in-tutorial-schema
                    https://trello.com/c/PQIls3Rp/34-changes-in-course-schema
                    https://trello.com/c/SFayipL1/32-changes-in-quiz-schema
                    https://trello.com/c/gCOYNlsC/35-changes-in-article-schema

### CODE REVIEW

File: server/DataBase/models/article.js [hunk 1]
```diff
@@ -7,19 +7,36 @@ let articleSchema = new mongoose.Schema({
...
+    title : {
         type : String,
         required : true,
         minlength : 1
     },
-    article : {type : String},
+    content : {type : String},
     comments : [{
         comment : {type : String},
         commentedBy : {type : String}
```
Comment (abdulhaqshah) [inline server/DataBase/models/article.js]: comment will have id and name

File: server/DataBase/models/article.js [hunk 2]
```diff
@@ -7,19 +7,36 @@ let articleSchema = new mongoose.Schema({
...
-    }]
+    }],
+    likes : [String],
```
Comment (abdulhaqshah) [inline server/DataBase/models/article.js:39]: like will have object of id and name

File: server/DataBase/models/tutorials.js [hunk 3]
```diff
@@ -6,28 +6,24 @@ let TutorialSchema = new mongoose.Schema({
...
-                type : String
-            }
-        }]
-    }],
-    active : {type : Boolean, default : false}
+    content : {
+        type : String
+    },
+    tags : [String],
+    category : String,
```
Comment (abdulhaqshah) [inline server/DataBase/models/tutorials.js]: {type : String} and follow this pattern every where

File: server/DataBase/models/tutorials.js [hunk 4]
```diff
@@ -6,28 +6,24 @@ let TutorialSchema = new mongoose.Schema({
...
+    createdAt : {type : Date, default : Date.now},
+    createdBy : String,
+    updatedAt : {type : Date, default : Date.now},
+    updatedBy : String,
```
Comment (abdulhaqshah) [inline server/DataBase/models/tutorials.js]: same as above

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

File: server/DataBase/models/article.js (hunk 1 above)
Comment (HarisShahidHameed) [reply server/DataBase/models/article.js]: Done

File: server/DataBase/models/article.js (hunk 2 above)
Comment (HarisShahidHameed) [reply server/DataBase/models/article.js:39]: Done

File: server/DataBase/models/tutorials.js (hunk 3 above)
Comment (HarisShahidHameed) [reply server/DataBase/models/tutorials.js]: Done

File: server/DataBase/models/tutorials.js (hunk 4 above)
Comment (HarisShahidHameed) [reply server/DataBase/models/tutorials.js]: Done

**Pull request merged**